tests/test_resources.py

```diff
@@ -49,5 +49,77 @@ def test_httpverb(self):
             response = method('foo')
             assert response.success
 
+    def test_pagination(self):
+      self.client.url = 'mock://api.com/{param}{?per_page}'
```

Comment:
You can try it. `public_gists_url: "https://api.github.com/gists/public"` doesn't advertise `per_page` but https://api.github.com/gists/public?per_page=1 works.